packages/wonder-blocks-dropdown/__snapshots__/generated-snapshot.test.js.snap

```diff
@@ -499,7 +499,7 @@ exports[`wonder-blocks-dropdown example 5 1`] = `
         "minWidth": 0,
         "padding": 0,
         "position": "relative",
-        "width": 100,
+        "width": "fit-content",
```

Comment:
Why did this change?

Reply:
because the opener wasn't getting truncated in that example, fixed now